carcara/src/ast/printer.rs

```diff
@@ -520,6 +520,11 @@ impl fmt::Display for Sort {
             Sort::Real => write!(f, "Real"),
             Sort::String => write!(f, "String"),
             Sort::RegLan => write!(f, "RegLan"),
+            Sort::Var(name) => write!(f, "{}", name),
+            Sort::ParamSort(args, s) => {
+                let par = format!("(par {:?} {})", args, s);
+                write!(f, "{}", par)
+            }
```

Comment:
You can pass these format arguments to `write!` directly, there's no need to first `format!` them into a string:
```suggestion
            Sort::ParamSort(args, s) => write!(f, "(par {:?} {})", args, s),
```